strikecircle/static/strikecircle/js/data_input_dashboard.js

```diff
@@ -1,4 +1,25 @@
 $(document).ready(function() {
+    const setCookie = (cname, cvalue, exdays) => {
+        var d = new Date();
+        d.setTime(d.getTime() + (exdays*24*60*60*1000));
+        var expires = "expires="+ d.toUTCString();
+        document.cookie = cname + "=" + cvalue + ";" + expires + ";path=/";
+    }
+
+    const getCookie = (cname) => {
```

Comment:
I'm guessing you already know this and that this was a stylistic choice, but just in case it wasn't -- arrow functions don't require you to put parentheses around the argument list if there's only one argument.